metadatastore/commands.py

```diff
@@ -671,24 +818,34 @@ def find_run_stops(run_start=None, **kwargs):
     -------
     run_stop : iterable of metadatastore.document.Document objects
     """
-    _format_time(kwargs)
-    # get the actual mongo document
     if run_start:
-        run_start = _get_mongo_document(run_start, RunStart)
-        kwargs['run_start_id'] = run_start.id
+        run_start_uid = _get_uid(run_start)
+        mongo_run_start = _find_documents(RunStart, uid=run_start_uid)
+        mongo_run_start = next(mongo_run_start)
```

Comment:
Perhaps the error that will be raised by this is not the *most useful error* possible, but seems reasonable to raise an error here.  If you pass in a `RunStart` document or a `run_start_uid` and we cannot find the mongo object that corresponds to that document or uid then something has gone very wrong. 

Feels like the correct thing to do to barf at this point, right?  What error should be raised here instead?